Documentation/contributing.rst

```diff
@@ -1183,60 +1398,110 @@ example patch that shows how this can be achieved.
++---------------------------------------+------------------------------------------------------------------+
+| `BETA-cilium-v1.1-K8s-Upstream`_      | Runs K8s upstream tests on merge into branch ``v1.1``            |
++---------------------------------------+------------------------------------------------------------------+
+| `BETA-cilium-v1.1-Docs`_              | Runs docs tests on merge into branch ``v1.1``                    |
++---------------------------------------+------------------------------------------------------------------+
+| `BETA-cilium-v1.1-Nightly`_           | Runs durability tests on branch ``v1.1`` every night             |
++---------------------------------------+------------------------------------------------------------------+
 
-    - 1.10 beta
-    - 1.11 alpha
+.. note::
 
-CI / Testing environment
-------------------------
+  ``BETA-cilium-v1.0-*`` is currently not subject to the daily triage process
+  as the quality of the tests backported to that branch does not justify the
+  effort.
+
+.. _Ginkgo-Tests-Validated-master: https://jenkins.cilium.io/job/cilium-ginkgo/job/cilium/job/master/lastBuild/
+.. _Ginkgo-Tests-Validated-1.0: https://jenkins.cilium.io/job/cilium-ginkgo/job/cilium/job/v1.0/lastBuild/
+.. _Ginkgo-CI-Tests-Pipeline: https://jenkins.cilium.io/job/Ginkgo-CI-Tests-Pipeline/
+.. _Master-Nightly-Tests-All: https://jenkins.cilium.io/job/Cilium-Master-Nightly-Tests-All/
+.. _Vagrant-Master-Boxes-Packer-Build: https://jenkins.cilium.io/job/Vagrant-Master-Boxes-Packer-Build/
+.. _github.com/cilium/packer-ci-build: https://github.com/cilium/packer-ci-build/
+.. _BETA-cilium-v1.1-standard: https://jenkins.cilium.io/view/BETA-Cilium-v1.1/job/BETA-cilium-v1.1-standard/
+.. _BETA-cilium-v1.1-K8s-all: https://jenkins.cilium.io/view/BETA-Cilium-v1.1/job/BETA-cilium-v1.1-K8s-all/
+.. _BETA-cilium-v1.1-K8s-Upstream: https://jenkins.cilium.io/view/BETA-Cilium-v1.1/job/BETA-cilium-v1.1-K8s-Upstream/
+.. _BETA-cilium-v1.1-Nightly: https://jenkins.cilium.io/view/BETA-Cilium-v1.1/job/BETA-cilium-v1.1-Nightly/
+.. _BETA-cilium-v1.1-Docs: https://jenkins.cilium.io/view/BETA-Cilium-v1.1/job/BETA-cilium-v1.1-Docs/
+
+Triage process
+^^^^^^^^^^^^^^
+
+#. Investigate the failure
+#. Search `GitHub issues <https://github.com/cilium/cilium/issues?utf8=%E2%9C%93&q=is%3Aissue+>`_
+   to see if bug is already filed. Make sure to also include closed issues in
+   your search as a CI issue can be considered solved and then re-appears.
+#. If no existing GitHub issues was found, file a new GitHub issue:
+
+   #. Attach zipfile downloaded from Jenkins with logs from failing test
+   #. If failure is a real bug:
+
+      #. Title: ``<Short bug description>``
+      #. Labels ``kind/bug`` and ``needs/triage``.
+   #. If failure is a CI test flake or if unsure:
+
+      #. Title ``CI: <testname>: <cause>``, e.g. ``CI: K8sValidatedPolicyTest Namespaces: cannot curl service``
+      #. Labels ``kind/bug/CI`` and ``needs/triage``
```

Comment:
Add to the `CI Flakes project`? https://github.com/cilium/cilium/projects/8

Reply:
Was not aware of this, added